src/components/engineStatusBanner/index.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019-2020 SURF.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import "./EngineStatusBanner.scss";
+
+import { EuiHeaderLink } from "@elastic/eui";
+import EngineSettingsContext from "contextProviders/engineSettingsProvider";
+import { useContext } from "react";
+import { FormattedMessage } from "react-intl";
+
+export default function EngineStatusBanner() {
+    const { engineStatus } = useContext(EngineSettingsContext);
+
+    return (
+        <EuiHeaderLink href="#">
+            <FormattedMessage id={`settings.status.engine.${engineStatus.global_status.toLocaleLowerCase()}`} />
```

Comment:
For readability I would extract engineStatus.global_status.toLocaleLowerCase() into a separate const, since it's also used in line 30